rules/build_sector.smk

```diff
@@ -782,7 +772,7 @@ rule prepare_sector_network:
         + "prenetworks/elec_s{simpl}_{clusters}_l{ll}_{opts}_{sector_opts}_{planning_horizons}.nc",
     threads: 1
     resources:
-        mem_mb=2000,
+        mem_mb=20000,
```

Comment:
that's quite a large increase, is that much memory necessary?

Reply:
that was actually not intended, thanks for pointing out